static/js/services/contact-summary.js

```diff
@@ -60,6 +60,7 @@ angular.module('inboxServices').service('ContactSummary',
     };
 
     return function(contact, reports, lineage) {
+      contact.parent = contact.parent || '';
```

Comment:
Shouldn't parent be an empty object rather than an empty string?

I'm a little concerned with this approach because it's only protecting a very specific case. If the script has `contact.parent.parent.parent` it will fail just as before. I wonder if it would just be better to document that not all contacts have parents so the script programmer knows to add checks.

In the longer term we want to have testing for configuration so this will be picked up earlier.

Reply:
I took an example of how district_hospitals parent field is populated in standard forms (an empty string).
For example: https://alpha.dev.medicmobile.org/medic/9a6741f5-304d-48a6-ac9b-0b9f6a041455.
I'm not the most knowledgeable person to decide if documenting is better.